qsim.cpp

```diff
@@ -311,6 +317,13 @@ Qsim::QemuCpu::QemuCpu(int id, const char* kernel, unsigned ram_mb,
   load_and_grab_pointers(get_qemu_lib(cpu_type).c_str());
 
   const char **cmd_argv = get_qemu_args(kernel, ram_mb, n_cpus, type, mode);
+
+  // print the Qemu args
+  /*
+  for(unsigned int i=0;cmd_argv[i]!=NULL;i++)
+      std::cout << "argv[" << i << "] = " << cmd_argv[i] << std::endl;
+  */
+
```

Comment:
Yes, these lines are also not very necessary, but it is hard to see how you exact start qemu. So maybe there are not so bad.

If you like I can delete them.